pkg/dotnet/core_deps/parse.go

```diff
@@ -19,6 +22,22 @@ func NewParser() types.Parser {
 	return &Parser{}
 }
 
+func splitNameVer(nameVer string) (name, version string) {
+	split := strings.Split(nameVer, "/")
+	if len(split) != 2 {
+		// Invalid name
+		log.Logger.Warnf("Cannot parse .NET library version from: %s", nameVer)
+		return "", ""
+	}
+	name = split[0]
+	version = split[1]
+	return
```

Comment:
Let's use same format(previously you used `return "", ""`)
```suggestion
	return split[0], split[1]
```